.github/workflows/close-pr.yml

```diff
@@ -0,0 +1,80 @@
+name: PR Tasks
+on:
+    pull_request:
+        types:
+            - closed
+
+permissions:
+    id-token: write
+    contents: read
+
+jobs:
+    deployment:
+        runs-on: ubuntu-latest
+        #environment: dev
+        steps:
+            - name: 'Az CLI login'
+              uses: azure/login@v2
+              with:
+                  client-id: ${{ secrets.AZURE_CLIENT_ID }}
+                  tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+                  subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+            - uses: Azure/get-keyvault-secrets@v1
+              with:
+                  keyvault: 'darts-portal-stg-kv'
+                  secrets: 'b2c-jenkins-tenant-id, b2c-jenkins-client-id, b2c-jenkins-client-secret'
+              id: bootstrapKv
+
+            - run: |
+                  mediaSignInUrl="https://darts-portal-pr-${PR_NUMBER}.dev.platform.hmcts.net/login/return"
+                  adminSignInUrl="https://darts-portal-pr-${PR_NUMBER}.dev.platform.hmcts.net/login/admin/return"
+                  adminPasswordResetUrl="https://darts-portal-pr-${PR_NUMBER}.dev.platform.hmcts.net/password-change-confirmation/true"
+                  mediaPasswordResetUrl="https://darts-portal-pr-${PR_NUMBER}.dev.platform.hmcts.net/password-change-confirmation/false"
```

Comment:
Is the idea that it checks these URLs exist as redirect URIs in the AD B2C app? If so, the ones we need to add are below, the paths used here aren't correct for the DARTS app.

```
https://darts-portal-pr-${PR_NUMBER}.dev.platform.hmcts.net/auth/callback
https://darts-portal-pr-${PR_NUMBER}.dev.platform.hmcts.net/auth/logout-callback
```